### PR TITLE
lib/model: Consistently show folder description in startup messages

### PR DESCRIPTION
**Pull request by calmh**
We missed a couple of places to show the folder description (id + label) instead of just the ID.

Since we anyway need the folderConfig for this I'm skipping the copying of all it's attributes that rwfolder did and just keeping the original around instead.

### CODE REVIEW

File: lib/model/rofolder.go [hunk 1]
```diff
@@ -20,6 +20,7 @@ func init() {
 
 type sendOnlyFolder struct {
 	folder
+	cfg config.FolderConfiguration
 }
```
Comment (calmh) [inline lib/model/rofolder.go]: Yeah, that'd be cleaner. And fix the tests. I'll get to it.

File: lib/model/rofolder.go [hunk 2]
```diff
@@ -20,7 +20,7 @@ func init() {
 
 type sendOnlyFolder struct {
 	folder
-	cfg config.FolderConfiguration
+	config.FolderConfiguration
 }
```
Comment (AudriusButkevicius) [inline lib/model/rofolder.go]: There was a joke about how Latvian president asked Lithuanian president to lend them tanks when Russian invaded Latvia.

The Lithuanian president asked whether the Latvian president would like one of them, or all of them... while Lithuania only had two...

TL;DR:
Given all of our two folders are embedding `folder`, I suggest we embed config in `folder` instead.

Comment (calmh) [inline lib/model/rofolder.go]: I have a funny joke about where exactly you can embed your config...

:trollface: anyway, that folder thing is a bit of a weird appending as it has almost no functionality. Should sort that out same day.

**Comment by AudriusButkevicius** [comment]
Fine.

**Comment by AudriusButkevicius** [comment]
@st-review merge

**Comment by st-review** [comment]
:ok_hand: Merged as 5c067661f4894fc386ed03f4e05ccd60370cad2a. Thanks, @calmh!
